packages/cli/src/build-ipfs-connection.util.ts

```diff
@@ -0,0 +1,66 @@
+import IPFS from "ipfs-core"
+
+import dagJose from 'dag-jose'
+// @ts-ignore
+import multiformats from 'multiformats/basics'
+// @ts-ignore
+import legacy from 'multiformats/legacy'
+import ipfsClient from "ipfs-http-client"
+import { IpfsApi } from "@ceramicnetwork/common"
+
+multiformats.multicodec.add(dagJose)
+const dagJoseFormat = legacy(multiformats, dagJose.name)
+
+const IPFS_DHT_SERVER_MODE = process.env.IPFS_DHT_SERVER_MODE === 'true'
+
+const BOOTSTRAP_LIST = [
+    '/dns4/ipfs-dev.3boxlabs.com/tcp/4012/wss/p2p/QmeRWpCaEpQLY1yJFFHf7pFbZZapXoj5E4pKvUtyExMUKk',
+    '/dns4/ipfs-dev.ceramic.network/tcp/4012/wss/p2p/QmadvKqYyLMHYjnwswd2bHM3UCFp4F4oSFdobYUKSPjBaZ',
+    '/dns4/ipfs-dev-cas.3boxlabs.com/tcp/4012/wss/p2p/QmPHLQoWhK4CMPPgxGQxjNYEp1fMB8NPpoLaaR2VDMNbcr',
+    '/dns4/ipfs-clay-cas.3boxlabs.com/tcp/4012/wss/p2p/QmQotCKxiMWt935TyCBFTN23jaivxwrZ3uD58wNxeg5npi',
+    '/dns4/ipfs-clay.ceramic.network/tcp/4012/wss/p2p/QmQqq1oVkwpJdKNdiWvrftgWE2nov4AFGPBauZ2AakxmoR',
+    '/dns4/ipfs-clay.3boxlabs.com/tcp/4012/wss/p2p/QmQotCKxiMWt935TyCBFTN23jaivxwrZ3uD58wNxeg5npi'
```

Comment:
Can we make sure to connect to the clay nodes only when `network` is configured to `testnet-clay`?

Reply:
Ah, see that you did something like this 👍 I imagine when we have the mainnet nodes up we will connect to a different set of nodes of that configuration :)

Reply:
So, as for now, all the nodes are still here @oed @valmack ?

Reply:
Clarification regarding network names: if the "bootstrap" nodes do not follow some other pubsub topic like "/testing/bugging", the nodes connected to the bootstrap ones would not hear from the topic, unless finally connected to each other.

Reply:
These three nodes we should connect to only if we are on `testnet-clay`.